spec/models/user_spec.rb

```diff
@@ -2,33 +2,50 @@
 
 
 RSpec.describe User, :type => :model do
-  let!(:valid_user) { User.create(first_name: "Alice", last_name: "Smith", email: "kit@kit.com") }
+  let!(:valid_user) { User.create(first_name: "Alice",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.